base58/src/lib.rs

```diff
@@ -24,13 +24,14 @@ extern crate std;
 
 static BASE58_CHARS: &[u8] = b"123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 
-use core::{fmt, iter, slice, str};
 #[cfg(not(feature = "std"))]
 pub use alloc::{string::String, vec::Vec};
+use core::{fmt, iter, slice, str};
 #[cfg(feature = "std")]
 pub use std::{string::String, vec::Vec};
 
 use hashes::{sha256d, Hash};
+use internals::impl_from_infallible;
```

Comment:
You could just use the fully qualified path if you want to then you don't have to touch import statements at all, may make your life easier.